test/drbgtest.c

```diff
@@ -1271,6 +1260,7 @@ static int test_set_defaults(void)
            && TEST_true(RAND_DRBG_uninstantiate(private));
 }
 
+#if 0
```

Comment:
Why? Similar instances below.

Reply:
The CRNG testing is done deep inside the provider and we've got no way to access it.  Setting the DRBG parent or callbacks bypasses the testing.

A full rework of the approach to the CRNG testing is the solution.  I'm thinking along the lines of a new random algorithm "CRNGT" which does the testing and which can be inserted into the chain of DRBGs created.  This requires that seed sources are also random algorithms and it likely requires another that multiplexes the seed material.  Maybe later.  Think very simple BIOs in a chain, except it is a tree not a chain.